### PR TITLE
Update travis php versions

### PR DESCRIPTION
**Pull request by vinkla**
Added PHP 7.1 and updated to stable version of 7.2.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-
 language: php
 
 php:
-  - '7.0'
-  - '7.2.0RC5'
+  - 7.0
+  - 7.1
```
Comment (felixfbecker) [inline .travis.yml]: do we really need to test on 7.1 if we test the lowest version 7.0 and the newest version 7.2? Worried about build time for no gain

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-
 language: php
 
 php:
-  - '7.0'
-  - '7.2.0RC5'
+  - 7.0
```
Comment (felixfbecker) [inline .travis.yml]: Please keep using strings. `7.0` == `7` == `7.x.x`, `'7.0'` == `7.0.x`

File: .travis.yml (hunk 1 above)
Comment (vinkla) [reply .travis.yml]: I've removed version 7.1 from the commit.

File: .travis.yml (hunk 2 above)
Comment (vinkla) [reply .travis.yml]: What? Didn't know about that. Do you know where I can find this in Travis documentation?

Comment (felixfbecker) [reply .travis.yml]: I don't think that is a Travis thing. In YAML if you don't quote `7.0` it's a floating point number so it is the same as `7`. Now whatever version manager Travis uses to install runtimes will get `7`, not `7.0`, so specifying `7.0` is at least misleading. I know that `nvm install 7` will install the latest `7.x.x` version, while `nvm install 7.0` will install the latest `7.0.x` version. I don't know how it is for PHP but there must have been a reason for this commit https://github.com/felixfbecker/php-language-server/commit/41e84880b380368ae5862c01e9a00d55067c48ec and there is no disadvantage to using strings.

Comment (vinkla) [reply .travis.yml]: Oh, I had no idea. Thanks for taking the time to explain.

**Pull request merged**